### ISSUE TITLE
[Feature]: document of `xqc_log`

### ISSUE THREAD

**Issue by Luffbee** [alibaba/xquic#206]
### Before request a new feature

- [X] I have searched the [issues](https://github.com/alibaba/xquic/issues) of this repository and believe that this is not a duplicate.

### Describe the feature you'd like supported

`xqc_log` use different format strings from `printf`, therefore, a document is needed.

**Comment by RuiqiZhou** [comment]
Does this document solve your problem? [Event_log module docs](https://github.com/alibaba/xquic/blob/main/docs/docs-zh/Event_log-zh.md)

If not, could you please describe your requirements in more detail

**Comment by Luffbee** [comment]
> 

No, I want a document describe the **format string** of `xqc_log`. For example, when I want to print a `uint32_t`, which format specifier should I use? I used `%u` once, but it's not working, so the format string is only **similar** to `printf`, I want to know the detail differences.

**Comment by RuiqiZhou** [comment]
> > 
> 
> No, I want a document describe the **format string** of `xqc_log`. For example, when I want to print a `uint32_t`, which format specifier should I use? I used `%u` once, but it's not working, so the format string is only **similar** to `printf`, I want to know the detail differences.

We follow the [Nginx Formatting](http://nginx.org/en/docs/dev/development_guide.html#formatting).
Maybe we should add this link to the document above?

**Comment by Luffbee** [comment]
This link is helpful, but I think it's still missing some details, for example, how should I print `uint64_t`? I looked [`src/core/ngx_string.c`](https://github.com/nginx/nginx/blob/f7ea8c76b55f730daa3b63f5511feb564b44d901/src/core/ngx_string.c#L90-L119) (mentioned in the link), it gives a better explanation. A comment or document like this will be good.
``` c
/*
 * supported formats:
 *    %[0][width][x][X]O        off_t
 *    %[0][width]T              time_t
 *    %[0][width][u][x|X]z      ssize_t/size_t
 *    %[0][width][u][x|X]d      int/u_int
 *    %[0][width][u][x|X]l      long
 *    %[0][width|m][u][x|X]i    ngx_int_t/ngx_uint_t
 *    %[0][width][u][x|X]D      int32_t/uint32_t
 *    %[0][width][u][x|X]L      int64_t/uint64_t
 *    %[0][width|m][u][x|X]A    ngx_atomic_int_t/ngx_atomic_uint_t
 *    %[0][width][.width]f      double, max valid number fits to %18.15f
 *    %P                        ngx_pid_t
 *    %M                        ngx_msec_t
 *    %r                        rlim_t
 *    %p                        void *
 *    %[x|X]V                   ngx_str_t *
 *    %[x|X]v                   ngx_variable_value_t *
 *    %[x|X]s                   null-terminated string
 *    %*[x|X]s                  length and string
 *    %Z                        '\0'
 *    %N                        '\n'
 *    %c                        char
 *    %%                        %
 *
 *  reserved:
 *    %t                        ptrdiff_t
 *    %S                        null-terminated wchar string
 *    %C                        wchar
 */
```
